MCHI/tp_dict.json

```diff
@@ -0,0 +1,2497 @@
+  "30": "30",
+  "31": "31",
+  "32": "32",
+  "33": "33",
+  "34": "34",
+  "35": "35",
+  "36": "36",
+  "37": "37",
+  "38": "38",
+  "39": "39",
+  "40": "40",
+  "41": "41",
+  "42": "42",
+  "43": "43",
+  "44": "44",
+  "45": "45",
+  "46": "46",
+  "47": "47",
+  "48": "48",
+  "49": "49",
+  "50": "50",
+  "51": "51",
+  "52": "52",
+  "53": "53",
+  "54": "54",
+  "55": "55",
+  "56": "56",
+  "57": "57",
+  "58": "58",
+  "59": "59",
+  "60": "60",
+  "61": "61",
+  "62": "62",
+  "63": "63",
+  "■アイコン優先順位チェック■": "Icon Priority Check",
+  "ID_INFO_ROOM_MAX_TEXEL": "ID_INFO_ROOM_MAX_TEXEL",
+  "ID_INFO_SWITCH": "ID_INFO_SWITCH",
+  "■デバック用マップ拡大■": "Map expansion for debugging",
+  "ONで拡大": "Enlarge on",
+  "■マップ表示位置■": "Map Display Location",
+  "ONで調整有効": "Adjustment enabled on",
+  "*Wide(16:9)": "*Wide(16:9)",
+  "*Normal(4:3)": "*Normal(4:3)",
+  "*GC(4:3) WiiでMAPアイコン無しの場合のY座標にも使用します": "*GC(4:3) Used for the Y coordinate in the case of no MAP icon on the Wii",
+  "■マップ表示サイズ■": "Map Display Size",
+  "・ON変更": "Change ON",
+  "■マップサイズ■": "Map Size",
+  "・サイズタイプの変更<サイズ変更後はテクスチャーサイズ変更をして下さい>": "Change size type <After changing size, please change the texture size>.",
+  "中": "中",
+  "・各サイズ<サイズ変更後はテクスチャーサイズ変更をして下さい>": "Each size < texture size change after size change>",
+  "・中": "Inside",
+  "・大": "Large",
+  "・小": "Small",
+  "・テクスチャー解像度半分": "Texture resolution half",
+  "テクスチャーサイズ変更": "Texture Size Change",
+  "■マップアルファ■": "Map Alpha",
+  "マップアルファテストONで有効": "Map Alpha Test ON enabled",
+  "■強制マップ上端変更■": "Forced map top change",
+  "ONからOFFにする場合は-1にしてからOFFにして下さい": "If you want to turn it off from ON, set it to -1 and then turn it off.",
+  "■マップ表示禁止■": "Map display prohibited",
+  "■マップレンダリング禁止■": "Map rendering prohibited",
+  "必ず最高速度ジャンプ": "Always jump to maximum speed",
+  "相撲待機旋回許可": "Sumo standby swivel permission",
+  "無敵状態": "Invincible state",
+  "距離減衰": "Distance Attenuation",
+  "減衰なし [GX_DA_OFF]": "No attenuation [GX_DA_OFF]",
+  "おだやか [GX_DA_GENTLE]": "Gentle [GX_DA_GENTLE]",
+  "中間     [GX_DA_MEDIUM]": "Intermediate [GX_DA_MEDIUM]",
+  "険しい   [GX_DA_STEEP]": "Rugged [GX_DA_STEEP]",
+  "角度減衰": "Angle Attenuation",
+  "ポイント [GX_SP_OFF]": "Point [GX_SP_OFF]",
+  "直角     [GX_SP_FLAT]": "Right Angle [GX_SP_FLAT]",
+  "COS曲線  [GX_SP_COS]": "COS curve [GX_SP_COS]",
+  "COS2曲線 [GX_SP_COS2]": "COS2 Curve [GX_SP_COS2]",
+  "シャープ [GX_SP_SHARP]": "Sharp [GX_SP_SHARP]",
+  "リング状 [GX_SP_RING1]": "Ring-shaped [GX_SP_RING1]",
+  "リング状2[GX_SP_RING2]": "Ring-shaped 2[GX_SP_RING2]",
+  "強制刺さり": "Forced Biting",
+  "警告消去": "Warning faded",
+  "ＢＭＰキャプチャー": "BMP Capture",
+  "スナップショット": "Snapshot",
+  "ＣＰＵ時間": "CPU Time",
+  "ホストＩＯ使用": "Host IO use",
+  "メーター表示(JUTProcBar)": "Meter Display (JUTProcBar)",
+  "プリント表示(JUTDbPrint)": "Print View (JUTDbPrint)",
+  "２Ｄ表示": "2D view",
+  "パーティクル表示": "Particle Display",
+  "パーティクル２５４固定": "Particle 254 fixed",
+  "パックアーカイブモード": "Pack Archive Mode",
+  "－－－－－－－【プリント】－－－－－－－": "-------【プリント】-- -----",
```

Comment:
nice

Reply:
deepl.com says this translates to "Print" fyi, no clue why the API bungled it up